stl/inc/ranges

```diff
@@ -411,31 +421,44 @@ namespace ranges {
     };
 
     // clang-format off
-    template <_Copy_constructible_object _Ty>
-        requires (copyable<_Ty>
-            || (is_nothrow_copy_constructible_v<_Ty>
-                && (movable<_Ty> || is_nothrow_move_constructible_v<_Ty>)))
-    class _Copyable_box<_Ty> { // provide the same API more efficiently when we can avoid the disengaged state
+    template <class _Ty>
+    concept _Use_simple_movable_box_wrapper =
+        // 1. If copy_constructible<T> is true, movable-box<T> should store only a T if either T models copyable, or

```

Comment:
This is quoting Standardese, so I think it should cite `[range.move.wrap]`, the new section. (If you cite paragraph numbers, we'll need to wait for a new Working Paper so we can cite its document number.)